docs/tutorials/add-environment-variable.md

```diff
@@ -1,32 +1,32 @@
+The environment variables in Appwrite are prefixed with `_APP_`. If it belongs to a specific category, the category name is appended as `_APP_REDIS` for the Redis category. The available categories are General, Redis, MariaDB, InfluxDB, StatsD, SMTP, Storage and Functions. Finally, a properly describing name is given to the variable. For example, `_APP_REDIS_HOST` is an environment variable for Redis connection host. You can find more information on available categories and existing environment variables in the [environment variables doc](https://appwrite.io/docs/environment-variables).
 
 ### Describe new environment variable
-First of all, we add the new environment variable to `app/config/variables.php` in the designated category. If none of the categories fit, add it to the General category. Copy the existing variables description to create a new one, so that you will not miss any required fields.
+First of all, we added the new environment variable to `app/config/variables.php` in the designated category. If none of the categories fit, add it to the General category. Copy the description of the existing variable to create a new one so that you will not miss any required fields.
 
 This information is also used to generate the website documentation at https://appwrite.io/docs/environment-variables, so please use good descriptions that clearly define the purpose and other required info about the environment variable that you are adding.
 
 ### Add to .env and Dockerfile
-If newly introduced environment variable has a default value, add it to the `.env` and `Dockerfile` along with other environment variables. `.env` file uses settings for Appwrite development environment.
+If the newly introduced environment variable has a default value, add it to the `.env` and `Dockerfile` along with other environment variables. `.env` file uses settings for Appwrite development environment.
 
-### Add to docker compose file and template
-Add the new environment variables to the `docker-compose.yml` and `app/views/install/compose.phtml` for each docker services that require access to those environment variables.
+### Add to docker-compose file and template
+Add the new environment variables to the `docker-compose.yml` and `app/views/install/compose.phtml` for each docker service that requires access to those environment variables.
 
-The `docker-compose.yml` file is used by the Appwrite maintainers during development where as `app/views/install/compose.phtml` file is used by the Appwrite setup script.
+The Appwrite maintainers use the `docker-compose.yml` file during development, whereas the `app/views/install/compose.phtml` file is used by the Appwrite setup script.
```

Comment:
> The Appwrite maintainers use the `docker-compose.yml` file during development
 
 The first edit here is actually not necessary and create an inconsistency in the voice of the verbs used throughout the line. It would be a good idea to revert back to the original here:
 
 > The `docker-compose.yml` file is used by the Appwrite maintainers during development

Reply:
Actually using active voice is much better in a docs compared to passive voice. If you allow me, I will copy-edit the entire document to active voice.

Reply:
That will not be necessary at the moment. For now, let's keep that sentence consistent with passive voice.